.buildkite/pipeline.yml

```diff
@@ -203,3 +203,16 @@ steps:
           image: "rustvmm/dev:v6"
           always-pull: true
           propagate-environment: true
+
+  - label: "cargo-audit"
+    commands:
+      - pytest rust-vmm-ci/integration_tests/test_cargo_audit.py
```

Comment:
We should run this with `-s` so that we can see what are the vulnerable crates in case there are any:
```pytest rust-vmm-ci/integration_tests/test_cargo_audit.py -s```